src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/transformer/ArrivalTransformer.kt

```diff
@@ -0,0 +1,19 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.transformer
+
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.health.api.model.Arrival
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.ArrivalEntity
+
+@Component
+class ArrivalTransformer() {
```

Comment:
We've found that a nice pattern is to add a constructor to the API class that accepts an Entity as the single parameter. Saves on yet another component to inject for a single use with little benefit (transformations like this are trivial).

Reply:
This is another case where generating the API models from the spec means we can't unfortunately.